### PR TITLE
Fix the telegun save

### PR DESCRIPTION
**Pull request by Ryozuki**

### CODE REVIEW

File: src/game/client/components/binds.cpp [hunk 1]
```diff
@@ -40,6 +40,9 @@ CBinds::~CBinds()
 
 void CBinds::Bind(int KeyID, const char *pStr, bool FreeOnly, int Modifier)
 {
+	if(!pStr)
+		return;
```
Comment (def-) [inline src/game/client/components/binds.cpp]: doesn't belong in this PR I guess

**Comment by Ryozuki** [comment]
it still doesn't work im out of ideas
